SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -2345,14 +2340,13 @@ void USpatialNetDriver::DelayedRetireEntity(Worker_EntityId EntityId, float Dela
 	}, Delay, false);
 }
 
-void USpatialNetDriver::HandleStartupOpQueueing(const TArray<Worker_OpList*>& InOpLists)
+void USpatialNetDriver::HandleStartupOpQueueing(TArray<SpatialGDK::OpList> InOpLists)
```

Comment:
Should we make this an explicit R value? Seems a mistake to allow a user to copy into this func.

Reply:
They both won't compile if you try and copy something. I'd guess one will say it can't bind an rvalue ref to an lvalue and the other will say that it can't find a copy constructor.

The reason I prefer by value for sink params is that it makes it clear that ownership is being transferred. The previous signature of a const ref was probably not correct but compiled because of how const propagates through pointers and how we did ownership.